views/tx.tmpl

```diff
@@ -53,110 +53,110 @@
+                        </td>
+                    </tr>
                 {{end}}
                 {{if eq .Type "Ticket"}}
-                {{if (and (.TicketInfo.VoteLuck) (eq .Mature "True"))}}
-                <tr>
-                    <td class="text-right pr-2 h1rem p03rem0">LUCK</td>
-                    <td>
-                        {{printf "%.4f" .TicketInfo.VoteLuck}}
-                        {{if (or (le .TicketInfo.TicketExpiry .Confirmations) (ne .TicketInfo.SpendStatus "Voted"))}} possible, {{.TicketInfo.LuckStatus}} {{else}} final, {{.TicketInfo.LuckStatus}} {{end}}
-                        (Best: {{.TicketInfo.BestLuck}} Avg: {{.TicketInfo.AvgLuck}})
-                    </td>
-                </tr>
-                {{end}}
-                {{if .TicketInfo.TicketExpiryDaysLeft}}
-                {{if eq .Mature "True"}}
-                {{if ge .TicketInfo.TicketExpiry .Confirmations}}
-                <tr>
-                    <td class="text-right pr-2 h1rem p03rem0">PROBABILITY</td>
-                    <td>
-                        <div class="row">
-                            <div class="col-11 col-lg-8 col-sm-12">
-                                <div class="progress" style="max-width: 330px">
-                                    <div
-                                        class="progress-bar"
-                                        role="progressbar"
-                                        style="width: {{.TicketInfo.Probability}}%;"
-                                        aria-valuenow="{{.TicketInfo.Probability}}"
-                                        aria-valuemin="0"
-                                        aria-valuemax="1"
-                                    >
-                                        <span class="nowrap pl-1 pr-1">
-                                            {{printf "%.2f%%" .TicketInfo.Probability}} Chance
-                                        </span>
+                    {{if  (eq .Mature "True")}}
+                        <tr>
+                            <td class="text-right pr-2 h1rem p03rem0">LUCK</td>
+                            <td>
+                                {{printf "%.4f" .TicketInfo.VoteLuck}}
+                                {{if (or (lt .TicketInfo.TicketExpiry .TicketInfo.ShortConfirms) (ne .TicketInfo.SpendStatus "Voted"))}} possible, {{.TicketInfo.LuckStatus}} {{else}} final, {{.TicketInfo.LuckStatus}} {{end}}
+                                (Best: {{.TicketInfo.BestLuck}} Avg: {{.TicketInfo.AvgLuck}})
+                            </td>
+                        </tr>
+                    {{end}}
+                    
+                    {{if and (and (eq .Mature "True")  (ne .TicketInfo.SpendStatus "Voted")) (eq .TicketInfo.PoolStatus "live")}}
+                        {{if ge .TicketInfo.TicketExpiry .TicketInfo.ShortConfirms}}
+                            <tr>
+                                <td class="text-right pr-2 h1rem p03rem0">PROBABILITY</td>
```

Comment:
I'm starting to think this would be more accessible to the user phrased like "Chance of expiry"